available_images.md

```diff
@@ -92,13 +92,14 @@ You can pin your version by adding the version tag to your URL as follows:
 |TensorFlow 1.15.3  |training, inference 	 |Training: Yes, Inference: No     |GPU 		| 2.7 (py27), 3.6 (py36)	|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.3-gpu-py36-cu100-ubuntu18.04  |v4.6           |
 |TensorFlow 1.15.3  |training 	             |Yes                        	   |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.3-cpu-py37-ubuntu18.04        |v7.2           |
 |TensorFlow 1.15.3  |training, inference 	 |Training: Yes, Inference: No     |GPU 		| 2.7 (py27), 3.6 (py36) 	|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.3-cpu-py36-ubuntu18.04        |v4.6           |
-|MXNet 1.6.0        |training            	 |Yes                       	   |GPU 		| 2.7 (py27)            	|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.6.0-gpu-py27-cu101-ubuntu16.04        |v3.7           |
-|MXNet 1.6.0        |training, inference 	 |Training: Yes, Inference: No 	   |GPU 		| 2.7 (py27), 3.6 (py36) 	|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.6.0-gpu-py36-cu101-ubuntu16.04        |v3.6           |
-|MXNet 1.6.0        |training, inference 	 |Training: Yes, Inference: No 	   |CPU 		| 2.7 (py27), 3.6 (py36) 	|763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-training:1.6.0-cpu-py36-ubuntu16.04              |v3.6           |
+|MXNet 1.7.0        |training            	 |Yes                       	   |GPU 		| 3.6 (py36)            	|763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-training:1.7.0-gpu-py36-cu101-ubuntu16.04       |v1.1           |
```

Comment:
looks like these are using us-west-2 instead of us-east-1 like the rest of the images here -- let's put that back

Reply:
done